sponsorship/forms.py

```diff
@@ -0,0 +1,20 @@
+from django import forms
+from .models import SponsorshipProfile
+
+class SponsorshipProfileForm(forms.ModelForm):
+    class Meta:
+        model = SponsorshipProfile
+        fields = [
+            'main_contact',
+            'additional_contacts',
```

Comment:
I think this can be excluded for the first pass. We may need to design a way to add additional contacts. For example, the current user may need to invite the additional contacts to register first before they can be added here. (eg this shouldn't be just the dropdown of existing users). So to simplfy the PR, perhaps exclude this.